tests/foreman/ui/test_registration.py

```diff
@@ -522,7 +523,7 @@ def test_positive_host_registration_with_non_admin_user(
     role = target_sat.cli.Role.info({'name': 'Register hosts'})
     target_sat.cli.User.add_role({'id': user.id, 'role-id': role['id']})
 
-    with Session(test_name, user=user.login, password=user_password) as session:
+    with target_sat.ui_session(test_name, user=user.login, password=user_password) as session:
```

Comment:
In other tests, you changed `target_sat` to `module_target_sat`, but here you are changing to `target_sat`. Why?